tests/unit/test_hyper_transformer.py

```diff
@@ -2246,7 +2246,7 @@ def test_update_transformers_no_field_transformers(self):
         instance = HyperTransformer()
         instance._fitted = False
         mock_transformer = Mock()
-        mock_transformer.get_supported_sdtype.return_value = ['datetime']
```

Comment:
It seems like this mock isn't doing anything since it was spelled wrong and the test still passed. We can probably just remove these lines and assert at the end that the config is unchanged